torchgeo/datasets/patternnet.py

```diff
@@ -77,7 +78,7 @@ class PatternNet(VisionClassificationDataset):
     url = "https://drive.google.com/file/d/127lxXYqzO6Bd0yZhvEbgIfz95HaEnr9K"
     md5 = "96d54b3224c5350a98d55d5a7e6984ad"
     filename = "PatternNet.zip"
-    directory = "images"
+    directory = "PatternNet/images"
```

Comment:
This should be os.path.join("PatternNet", "images")) so it can work cross platform.

Reply:
Is this change necessary to get the tests working or necessary because the zipfile downloaded from Google Drive has this directory and we weren't correctly handling it?

Reply:
Without this change, I get a No such file error when downloading the dataset with download=True. The dataset is less than 2GB large, so you can also try yourself.